### PR TITLE
Add doubly, circular Linked List, and kruskal graphs

### PR DESCRIPTION
**Pull request by v007rj**
Added Code for Kruskal Algorithm in Graph section.

### CODE REVIEW

**Review by akrish4 — APPROVED** [review]

**Review by akrish4 — APPROVED** [review]

File: graphs/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Linked Lists
```
Comment (ming-tsai) [inline graphs/README.md:1]: Why the title is Linked Lists, but the folder is graphs

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Why the title is Linked Lists, but the folder is a graph, and the readme is the same as the linked list?

**Comment by v007rj** [comment]
I will update it soon.

**Review by v007rj — COMMENTED** [review]
Please accept

**Review by ming-tsai — APPROVED** [review]

**Comment by ming-tsai** [comment]
Nice work

**Pull request merged**